src/ingestion/ec2_describe_instances.py

```diff
@@ -114,12 +116,18 @@ def get_data_worker(account):
                 instances.extend(region)
             except Exception as e:
                 print(f"ec2_describe_instances: account: {account[1]} exception: {e}")
+                db.insert(AWS_ACCOUNTS_INFORMATION_TABLE, values=[(datetime.datetime.utcnow(), account[0], account[1], None, json_dumps(e))])
                 return None
         instance_list = [json.dumps({**instance,"AccountId":account[0]}, default=str) for instance in instances]
+        try:
+            db.insert(AWS_ACCOUNTS_INFORMATION_TABLE, values=[(datetime.datetime.utcnow(), account[0], account[1], len(instance_list), None)])
+        except Exception:
+            print('Failed to insert into AWS_ACCOUNT_INFORMATION table.')
         print(f"ec2_describe_instances: account: {account[1]} instances: {len(instance_list)}")
         return instance_list
     except Exception as e:
         print(f"ec2_describe_instances: account: {account[1]} exception: {e}")
+        db.insert(AWS_ACCOUNTS_INFORMATION_TABLE, values=[(datetime.datetime.utcnow(), account[0], account[1], None, json_dumps(e))])
```

Comment:
formatting how something is inserted (i.e. `json_dumps`) should be something this code does not concern itself with -- can we put it somewhere into db?

Reply:
Seems reasonable. I added Exception to the tuple of things which are jsony in `db.insert()`, so it'll json_dumps them the same way it does when you pass it a dict, list, or tuple.